firebase-installations-interop/src/main/java/com/google/firebase/installations/internal/HandleFidListener.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.installations.internal;
+
+/**
+ * Interface for un-registering a previously registered listener.
+ *
+ * @hide
+ */
+public interface HandleFidListener {
+
+  /**
+   * Unregisters a previously registered {@link FidListener}.
+   *
+   * @hide
+   */
+  void unregister(FidListener fidListener);
```

Comment:
The handle is supposed to represent a registration so it shouldn't take a listener as a parameter. For an example see
https://github.com/firebase/firebase-android-sdk/blob/c7d3073d6fe2c168d93acfac88c45ec49a583601/firebase-firestore/src/main/java/com/google/firebase/firestore/ListenerRegistration.java#L17-L25

Reply:
PTAL. Thanks